File: 2576-find-the-maximum-number-of-marked-indices/2576-find-the-maximum-number-of-marked-indices.cpp
class Solution {
public:
    int maxNumOfMarkedIndices(vector<int>& nums) {
        
        sort(nums.begin(),nums.end());
        
        int j = nums.size()-1;
        int count = 0;
        
        for(int i = nums.size()/2-1; i>=0;  i--){
            
            if(nums[i]*2 <= nums[j]){
                count+=2;
                j--;
            }
        }
        
        return count;
    }
};

File: 2576-find-the-maximum-number-of-marked-indices/2576-find-the-maximum-number-of-marked-indices.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxNumOfMarkedIndices = function(nums) {
    
    nums.sort((a,b)=> a - b);
    
    let j = nums.length-1;
    let count = 0;
    for(let i = Math.floor(nums.length/2) - 1; i>=0; i--){
        if(nums[i]*2 <= nums[j]){
            j--;
            count+=2;
        }
    }
    return count;
};